docs/source/contributing.rst

```diff
@@ -0,0 +1,71 @@
+Undebt_: Contributing
+==========================
+
+.. _Undebt: index.html
+.. default-role:: code
+
+Getting Started
+---------------
+
+Undebt's development is taking place on Github, so please go ahead and `fork`_ the repository if you want to begin contributing.
+
+.. _`fork`: https://github.com/Yelp/undebt#fork-destination-box
+
+You'll then want to get a local copy of the code base:
+
+.. code-block:: bash
+
+    git clone git@github.com:<your-username>/undebt.git
+
+Getting Setup
+-------------
+
+It is highly recommended that you firstly create a `virtual environment`_.
+
+.. _`virtual environment`: http://docs.python-guide.org/en/latest/dev/virtualenvs/
+
+You can then install the project dependencies:
+
+.. code-block:: bash
+
+    pip install -r requirements-dev.txt
+
+Running the Tests
+-----------------
+
+Undebt uses `tox`_ for testing.
+
+.. _`tox`: https://tox.readthedocs.io/en/latest/
+
+You can run the entire test suite:
+
+.. code-block:: bash
+
+    tox
+
+Or, run an individual environment:
+
+.. code-block:: bash
+
+    tox -e py35
+
+Note. If you do not have the required dependencies for each Tox environment, you will receive an error.
+Avoid this by passing the `--skip-missing-interpreters` option.
+
+Adding Documentation
```

Comment:
Soon, we should add a note here that the CLI docs are generated automatically, so you don't have to update the usage when you change the CLI.